config/Mekanism/general.toml

```diff
@@ -46,10 +46,10 @@
 	[general.dynamic_tank]
 		#Amount of fluid (mB) that each block of the dynamic tank contributes to the volume. Max = volume * fluidPerTank
 		#Range: 1 ~ 368224
-		fluidPerTank = 350000
+		fluidPerTank = 8000
 		#Amount of chemical (mB) that each block of the dynamic tank contributes to the volume. Max = volume * chemicalPerTank
 		#Range: 1 ~ 1581510980256305
-		chemicalPerTank = 16000000
+		chemicalPerTank = 8000
```

Comment:
Why are these being adjusted? We redid their values in 10.1.0 and I am relatively happy with their updated values.